DashAI/back/tasks/base_task.py

```diff
@@ -14,6 +14,27 @@ class BaseTask:
     def schema(self) -> Dict[str, Any]:
         raise NotImplementedError
 
+    @classmethod
+    def get_metadata(cls) -> Dict[str, Any]:
+        """Get metadata values for the current task
+
+        Returns:
+            Dict[str, Any]: Dictionary with the metadata
+        """
```

Comment:
Maybe add here a short description of what the metadata consists of (input/output types, cardinality)?